### ISSUE TITLE
FYI: No longer works on Ubuntu 22.04 (Daily Build)

### ISSUE THREAD

**Issue by vtpoet** [thiggy01/change-gdm-background#31]
Just a heads up; and realizing this is very early Beta.

**Comment by Caligatio** [comment]
With 22.04 now out as a stable release, it would be great to get this added! It looks like a trivial change on top of the impish support but impish support was removed without a useful comment in the commit.

**Comment by mendhak** [comment]
Yes, I was wondering about that. I do have a [PR open](https://github.com/thiggy01/change-gdm-background/pull/32), it was against the impish commit, which has been reverted since.  

I have a branch that works against Ubuntu 22.04 if you want, https://github.com/mendhak/change-gdm-background/tree/ubuntu-jammy  but would be great to get some insight from @thiggy01 !

**Comment by mendhak** [comment]
There's also [an answer on StackOverflow](https://askubuntu.com/a/1396066/26436) showing how to change the wallpaper without relying on this script. I did test it on Ubuntu 22.04 and 20.04 just now, seems ok. 

First, `sudo apt install systemd-container`

Then you can change the wallpaper using:

    sudo machinectl shell gdm@ /bin/bash -c "gsettings set com.ubuntu.login-screen background-picture-uri 'file:///usr/share/backgrounds/warty-final-ubuntu.png'"
